aksel.nav.no/website/app/(routes)/(designsystemet)/_ui/icon-page/IconPage.icondetails.tsx

```diff
@@ -75,79 +74,84 @@ function IconPageIconDetails({
     });
   }
 
-  const HeaderComponent = !hideModal ? Modal.Header : "div";
-
   const dedupedKeywords = new Set(metaData?.keywords ?? []);
 
-  return (
-    <div className={styles.iconDetails} data-inline>
-      {hideModal && (
-        <div className={styles.iconDetailsShowcase}>
-          <IconComponent fontSize="2rem" />
-        </div>
-      )}
-      <VStack gap="space-24" className={styles.iconDetailsContent}>
-        <HeaderComponent>
-          {!hideModal && (
-            <Box marginBlock="space-0 space-8">
-              <IconComponent aria-hidden fontSize="2rem" />
-            </Box>
-          )}
-          <Heading level="2" size="small">
-            {iconName}
-          </Heading>
-          <HStack gap="space-2">
-            <Icons.ArrowDownRightIcon fontSize="1.5rem" aria-hidden />
-            <BodyShort>{metaData?.sub_category}</BodyShort>
-          </HStack>
-          <HStack gap="space-8" marginBlock="space-12 0" as="ul">
-            {[...dedupedKeywords.keys()].map((keyword) => (
-              <li key={keyword}>
-                <Tag size="small" variant="neutral-moderate">
-                  {keyword}
-                </Tag>
-              </li>
-            ))}
-          </HStack>
-        </HeaderComponent>
-        <div className={styles.iconDetailsActions}>
-          <HStack gap="space-8">
-            {iconSvg && (
-              <Button as="a" href={downloadLink} download={`${iconName}.svg`}>
-                Last ned SVG
-              </Button>
-            )}
+  const DialogActions = () => (
```

Comment:
It's bad practice to define a component inside a component.

```suggestion
  const dialogActions = (
```